src/pepr/values.yaml

```diff
@@ -7,9 +7,34 @@ watcher:
   envFrom:
     - secretRef:
         name: uds-operator-config
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - pepr
```

Comment:
I'm not sure this is actually working the way we'd want it to? The label used here doesn't show up on the admission pods from what I can tell:
```yaml
  labels:
    app: pepr-uds-core
    pepr.dev/controller: admission
    pod-template-hash: f594d9cd4
    zarf-agent: patched
```

We probably want to key off of `app: pepr-uds-core`?